tests/Makefile

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
We already have two ways to trigger tests (BTest and via CMake where the former also executes the latter), and introducing another way to rum them will it easier to miss one (especially since the files in this directory makes it look like just running `btest` should be sufficient).

I would much prefer if we'd just add BTest which 
- checks that we build with the analyzers with a `@TEST-REQUIRES: have-zeek-plugin` (I think that's missing below),
- creates a build directory for the analyzer build,
- configures the analyzer build in a way that it detects an build-directory or installed Spicy to support both installation tests as well,
- builds the analyzers in their build directory,
- executes the analyzer tests from that BTest with `btest -c <ANALYZER_PATH>/tests/btest.cfg -j 5` (we don't have a way to access the current BTest instance `-j` value, so just use something smallish for CI; currently we just have 3 analyzers).

This should also avoid a lot of the curn this introduces in e.g., the setup.

We'll probably still need something in `.cirrus.yml` to archive the package BTest artifacts.

Reply:
As discussed, recursive btest calls don't work very well.